src/openrct2/world/Duck.cpp

```diff
@@ -333,27 +325,27 @@ void create_duck(const CoordsXY& pos)
     }
     duck->sprite_direction = direction << 3;
     duck->MoveTo({ targetPos.x, targetPos.y, 496 });
-    duck->state = DUCK_STATE::FLY_TO_WATER;
+    duck->state = Duck::DuckState::FlyToWater;
     duck->frame = 0;
 }
 
 void Duck::Update()
 {
-    switch (static_cast<DUCK_STATE>(state))
+    switch (static_cast<DuckState>(state))
```

Comment:
The cast seems unnecessary, isn't state the enum type already?

Reply:
Yes. Thank you for reviewing!